lib/server.js

```diff
@@ -165,11 +168,23 @@ server.get('/tasks/:task/stats', function(req, res, next) {
  */
 server.get('/tasks/:task/items', function(req, res, next) {
   // TODO: allow lock=locked and lock=unlocked when listing items
-  TaskItems.findAll({
+  const search = {
     where: {
       task_id: req.params.task
     }
-  })
+  };
+  if (req.query.lock) {
+    if (req.query.lock !== 'locked' && req.query.lock !== 'unlocked') {
```

Comment:
Should these strings go in https://github.com/osmlab/to-fix-backend/blob/postgis/lib/constants.js ? Not sure what's better here / what pattern we want to follow.

Reply:
@batpad I agree, but I looked at code here https://github.com/osmlab/to-fix-backend/blob/c19dcc7e0cca3a082004f6c96a21f739f5685e60/lib/server.js#L307 .

Reply:
I am okay both ways. But ya consistency across the code would nice to have.

Reply:
Moving these to constants would be cool with me. They probs won't change though, so I'm not too worried about it.